apps/examples/testcase/le_tc/kernel/tc_libc_signal.c

```diff
@@ -318,6 +318,35 @@ static void tc_libc_signal_raise(void)
 	TC_SUCCESS_RESULT();
 }
 
+/**
+* @fn                   :tc_libc_signal_psignal
+* @brief                :tc_libc_signal_psignal test sigpsignal function
+* @scenario             :If pinfo == NULL, it set errno EINVAL.
+*                        Else if message is not a null pointer and is not the empty string, the string pointed to by the message argument will be written, followed by a colon and a space.
+*                        Then the signal description string associated with signum will be written, followed by a newline.
+* API's covered         :psiginfo()
+* Preconditions         :none
+* Preconditions         :none
+* @return               :void
+*/
+
+static void tc_libc_signal_psignal(void)
+{
+	siginfo_t siginfo;
+	siginfo.si_signo = SIG1;
+
+	psiginfo(NULL, NULL);
+	TC_ASSERT_EQ("psignal", get_errno(), EINVAL);
+
+	psiginfo(&siginfo, "User-defined signal 1 (POSIX)");
```

Comment:
Please remain the expected result for comment.

Reply:
modify